src/datasource/datasource.js

```diff
@@ -23,12 +23,12 @@ export class DataSource {
     async getPocketInstance() {
         if (!this.pocket || !this.pocket.rpc()) {
 
-            const pocketPrivateKey = 'e56bd9d4205d95850a9646421eb940767061b19ae840231216296cc3f06349cfd87bad4cc26a6a0493421a446e6d2ebc86eaff074e971adea1a5187602b39a9e'
-            const pocketPublicKey = 'd87bad4cc26a6a0493421a446e6d2ebc86eaff074e971adea1a5187602b39a9e'
-            const pocketAddress = 'dec6ba13f0b2f86909291988b7b44db7db79fc5e'
-            const pocketPassphrase = 'poktDivineJustice4190!'
+            const pocketPrivateKey = '657773c03e9b11474d0e9e42c635b1591fff148902642f9251e11944309f83a5e0282373b32f617ffad172cac45e55df26a3f1d01beaa52cc262d523f25afe29'
```

Comment:
This needs to be a priority to remove all of these test keys from code.

Reply:
Please change these to environment variables, like POCKET_PRIVATE_KEY and POCKET_PASSPHRASE

Then include them with .dotenv -- process.ENV('POCKET_PRIVATE_KEY')

Do not commit the .env file you use for development.

If the environment variables don't exist at runtime, an error should be displayed.